internal/testing/conformance/kustomization.yaml

```diff
@@ -29,3 +29,10 @@ patches:
         value: {'consul.hashicorp.com/connect-inject': 'true', 'consul.hashicorp.com/connect-service-port': '3000'}
     target:
       kind: Deployment
+  # We don't have enough resources in the GitHub-hosted Actions runner to support 2 replicas
```

Comment:
Turns out kind runs out of CPU before all of the conformance testing resources can spin up. There's no easy option for increasing the size of the runner today. It sounds like the consul team is running into similar issues that we may need to figure out in the future if/when upstream adds more deployments.